regexp.go

```diff
@@ -183,8 +183,9 @@ func (r *routeRegexp) url(values map[string]string) (string, error) {
 // For a URL with foo=bar&baz=ding, we return only the relevant key
 // value pair for the routeRegexp.
 func (r *routeRegexp) getUrlQuery(req *http.Request) string {
+	blank := ""
```

Comment:
This change hurts the readability of the code.  The variable `blank` is used as a constant. Even if the declaration is changed to `const blank = ""`, the change hurts readability. It's one more line of code to understand with no extra value.